aspnetcore/signalr/configuration.md

```diff
@@ -211,7 +211,7 @@ Additional options for configuring timeout and keep-alive behavior are available
 | .NET Option | JavaScript Option | Description |
 | ----------- | ----------------- | ----------- |
 | `ServerTimeout` | `serverTimeoutInMilliseconds` | Timeout for server activity. If the server hasn't sent any message in this interval, the client considers the server disconnected and trigger the `Closed` event (`onclose` in JavaScript). |
-| `HandshakeTimeout` | Not configurable | Timeout for initial server handshake. If the server doesn't send a handshake response in this interval, the client cancels the handshake and trigger the `Closed` event (`onclose` in JavaScript). |
+| `HandshakeTimeout` | Not configurable | Timeout for initial server handshake. If the server doesn't send a handshake response in this interval, the client cancels the handshake and trigger the `Closed` event (`onclose` in JavaScript). This is an advanced setting that should only be modified if the server or network are particularly slow, causing handshake timeout errors. For more detail on the Handshake process, see the [SignalR Hub Protocol Specification](https://github.com/aspnet/SignalR/blob/master/specs/HubProtocol.md) |
```

Comment:
* If the comma before "causing" is dropped above, drop it here, too.
* Needs a period at the end (after the *SignalR Hub Protocol Specification* link).

btw- There's a subject-verb agreement problem here that predates your PR ...

> the client cancels the handshake and trigger

... it should be "triggers"